keyboards/aki27/cocot46plus/cocot46plus.c

```diff
@@ -0,0 +1,286 @@
+/*
+Copyright 2022 aki27
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "cocot46plus.h"
```

Comment:
```suggestion
```

Reply:
Other keyboards explicitly include their header file. Why is this not needed here?

Reply:
`QMK_KEYBOARD_H` will pull in that header, so its completely pointless.

Reply:
the `QMK_KEYBOARD_H` include should be removed instead of this one, as the header file name is known.

Reply:
> the QMK_KEYBOARD_H include should be removed instead of this one, as the header file name is known.

This is incorrect, its required for the generated keycodes.